src/main/java/org/sahagin/runlib/external/adapter/fluentlenium/FluentLeniumAdapter.java

```diff
@@ -26,16 +26,16 @@ public String getName() {
     }
 
     // can set null
-    public static void setAdapter(Fluent fluent) {
+    public static void setAdapter(FluentDriver fluent) {
         AdapterContainer container = AdapterContainer.globalInstance();
```

Comment:
@hiroshitoda Can this  break backward compatibility? If so, I will notice about this in the next release note.